engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/StdlibRuntimeServerTest.scala

```diff
@@ -0,0 +1,205 @@
+        .allowAllAccess(true)
+        .option(
+          RuntimeOptions.PACKAGES_PATH,
+          toPackagesPath(pkg.root.getAbsolutePath, stdlib.toString)
+        )
+        .option(RuntimeOptions.LOG_LEVEL, "WARNING")
+        .option(RuntimeOptions.INTERPRETER_SEQUENTIAL_COMMAND_EXECUTION, "true")
+        .option(RuntimeServerInfo.ENABLE_OPTION, "true")
+        .out(out)
+        .serverTransport { (uri, peer) =>
+          if (uri.toString == RuntimeServerInfo.URI) {
+            endPoint = peer
+            new MessageEndpoint {
+              override def sendText(text: String): Unit = {}
+
+              override def sendBinary(data: ByteBuffer): Unit =
+                Api.deserializeResponse(data).foreach(messageQueue.add)
+
+              override def sendPing(data: ByteBuffer): Unit = {}
+
+              override def sendPong(data: ByteBuffer): Unit = {}
+
+              override def sendClose(): Unit = {}
+            }
+          } else null
+        }
+        .build()
+    )
+    executionContext.context.initialize(LanguageInfo.ID)
+
+    def toPackagesPath(paths: String*): String =
+      paths.mkString(ContextPathSeparator)
+
+    def writeMain(contents: String): File =
+      Files.write(pkg.mainFile.toPath, contents.getBytes).toFile
+
+    def writeFile(file: File, contents: String): File =
+      Files.write(file.toPath, contents.getBytes).toFile
+
+    def writeInSrcDir(moduleName: String, contents: String): File = {
+      val file = new File(pkg.sourceDir, s"$moduleName.enso")
+      Files.write(file.toPath, contents.getBytes).toFile
+    }
+
+    def send(msg: Api.Request): Unit = endPoint.sendBinary(Api.serialize(msg))
+
+    def receiveNone: Option[Api.Response] = {
+      Option(messageQueue.poll())
+    }
+
+    def receive: Option[Api.Response] = {
+      Option(messageQueue.poll(3, TimeUnit.SECONDS))
+    }
+
+    def receive(timeout: Long): Option[Api.Response] = {
+      Option(messageQueue.poll(timeout, TimeUnit.SECONDS))
+    }
+
+    def receiveN(n: Int): List[Api.Response] = {
+      Iterator.continually(receive).take(n).flatten.toList
+    }
+
+    def receiveN(n: Int, timeout: Long): List[Api.Response] = {
+      Iterator.continually(receive(timeout)).take(n).flatten.toList
+    }
+
+    def consumeOut: List[String] = {
+      val result = out.toString
+      out.reset()
+      result.linesIterator.toList
+    }
+
+    def executionSuccessful(contextId: UUID): Api.Response =
+      Api.Response(Api.ExecutionSuccessful(contextId))
+
+  }
+
+  override protected def beforeEach(): Unit = {
+    context = new TestContext("Test")
+    val Some(Api.Response(_, Api.InitializedNotification())) = context.receive
+  }
+
+  val ModulesInScope = Seq(
+    "Base.Bench_Utils",
+    "Base.Boolean",
+    "Base.List",
+    "Base.Main",
+    "Base.Maybe",
+    "Base.Process.Exit_Code",
+    "Base.Process",
+    "Base.System.Platform",
+    "Base.Test",
+    "Base.Vector",
+    "Test.Main"
```

Comment:
This seems brittle as we add new modules to the stdlib.

Reply:
You're right, it might be quite annoying to fix this test every time we introduce a new module. I will reimplement this check :+1: